src/main/java/uk/gov/justice/hmpps/prison/api/resource/AdjudicationsResource.java

```diff
@@ -72,4 +80,20 @@ public AdjudicationDetail getAdjudication(
     public List<AdjudicationDetail> getAdjudications(@ApiParam(value = "The adjudication numbers", required = true, example = "[1,2,3]") @RequestBody final List<Long> adjudicationNumbers) {
         return adjudicationsService.getAdjudications(adjudicationNumbers);
     }
+
+    @ApiOperation(value = "Search of adjudications", notes = "Requires MAINTAIN_ADJUDICATIONS access")
+    @ApiImplicitParams({
+        @ApiImplicitParam(name = "page", dataType = "java.lang.Integer", paramType = "query",
+            value = "Results page you want to retrieve (0..N). Default 0, e.g. the first page", example = "0"),
+        @ApiImplicitParam(name = "size", dataType = "java.lang.Integer", paramType = "query",
+            value = "Number of records per page. Default 10"),
+        @ApiImplicitParam(name = "sort", dataType = "java.lang.String", paramType = "query",
+            value = "Sort as combined comma separated property and uppercase direction. Multiple sort params allowed to sort by multiple properties. Default to incidentDate,incidentTime ASC")})
+    @PostMapping("/search")
+    @PreAuthorize("hasRole('MAINTAIN_ADJUDICATIONS')")
```

Comment:
Although we are using this to get a users reported adjudications of a particular agency, I suspect we will need to do something like this for other calls, and so I thought a generic name like search was best here